internal_transfers/actions/tests/e2e/getInternalImbalance.spec.ts

```diff
@@ -0,0 +1,157 @@
+import { getInternalImbalance, MinimalTxData } from "../../src/accounting";
+import { TenderlySimulator } from "../../src/simulate/tenderly";
+import { ethers } from "ethers";
+
+const simulator = new TenderlySimulator(
+  process.env["TENDERLY_USER"] || "INVALID_USER",
+  process.env["TENDERLY_PROJECT"] || "TENDERLY_PROJECT",
+  process.env["TENDERLY_ACCESS_KEY"] || "TENDERLY_ACCESS_KEY"
+);
+
+async function TxDataFromHash(txHash: string): Promise<MinimalTxData> {
+  const provider = ethers.getDefaultProvider(
+    process.env["NODE_URL"] || "NODE_URL"
+  );
+  const transaction = await provider.getTransactionReceipt(txHash);
+  if (transaction === null) {
+    throw new Error(`invalid transaction hash ${txHash} - try again`);
+  }
+  const { from, hash, logs } = transaction;
+  const tenderlyLogs = logs.map((log) => {
```

Comment:
I can't redefine a constant `logs` that already exists. Open to another name though... but it is quite specific to tenderly since this is an e2e test and I am explicitly using a `tenderlySimulator`.

Reply:
My proposal would be:

```js
const { from, hash, logs: readonlyLogs } = transaction;
const logs = readonlyLogs.map((log) => {
```

This imho makes it clearer why you have to transform the logs and reinforces the comment that appears later.